chunky/src/java/se/llbit/chunky/ui/CreditsController.java

```diff
@@ -98,16 +103,24 @@ public void initialize(URL location, ResourceBundle resources) {
     ghContributors.setOnAction(e -> launchAndReset(ghContributors, "https://github.com/chunky-dev/chunky/graphs/contributors"));
 
     markdown.setBorder(Border.EMPTY);
-    markdown.setOnAction(e -> launchAndReset(markdown, "https://github.com/chunky-dev/chunky/blob/master/licenses/Markdown.txt"));
-
-    apacheLicense.setBorder(Border.EMPTY);
-    apacheLicense.setOnAction(e -> launchAndReset(apacheLicense, "http://www.apache.org/licenses/LICENSE-2.0.txt"));
+    markdown.setOnAction(e -> launchAndReset(markdown, "https://daringfireball.net/projects/markdown/"));
+    markdownLicense.setBorder(Border.EMPTY);
+    markdownLicense.setOnAction(e -> launchAndReset(markdownLicense, "https://daringfireball.net/projects/markdown/license"));
```

Comment:
Using markdown for the changelogs in the launcher would be great. But right now I don't even see where it is used :thinking: